plugins/queue/smtp_forward.js

```diff
@@ -62,6 +60,14 @@ exports.get_config = function (connection) {
     return plugin.cfg[dom];
 };
 
+exports.is_outbound_enabled = function (cfg) {
+    const plugin = this;
+
+    if ('enable_outbound' in cfg) return cfg.enable_outbound === 'true'; // pick up per-domain flag if set
```

Comment:
Are you sure this will do what you want? The in operator will fire if enable_outbound is defined in the cfg object. So you're going to return the string "true", even if the value of enable_outbound is boolean false. Shouldn't we be consistently using JS booleans instead of strings?

Reply:
@msimerson The config values in subdomains are not parsed out as booleans. Is there a way to specify this in the `booleans` section when reading the config file?

Reply:
Oh, I see `*.enable_tls`. Maybe I can do this for enable_outbound. trying now.